### ISSUE TITLE
Import from drivvo

### ISSUE THREAD

**Issue by AlfHou** [akhilrex/hammond#21]
It would be nice to be able to import data from the app [drivvo](https://www.drivvo.com/en).

**Comment by akhilrex** [comment]
Does it support export? If it does can you please export and share the data. You can anonymize any identifying information.

**Comment by Dyberg** [comment]
I've just added all my entries from Drivvo to Hammond, manual though.
You can only export the data if you got Drivvo Pro.

I've attached my export without any edits (Note that the language is Danish).
The translation is here:
ODOmeter (km),"Date","Fuel","Price / L","Total expenses.","Volume","Full tank!","Second fuel","Price / L","Total expenses.","Volume","Full tank!" 2,"Third fuel","Price / L","Total expenses.","Volume","Full tank!"
[fuel.zip](https://github.com/akhilrex/hammond/files/6598416/fuel.zip)

**Comment by AlfHou** [comment]
> You can only export the data if you got Drivvo Pro.
True, but I believe this is against GDPR, so I've sent an email to the drivvo team, requesting my data. Still waiting for a response

**Comment by akhilrex** [comment]
> I've just added all my entries from Drivvo to Hammond, manual though.
> You can only export the data if you got Drivvo Pro.
> 
> I've attached my export without any edits (Note that the language is Danish).
> The translation is here:
> ODOmeter (km),"Date","Fuel","Price / L","Total expenses.","Volume","Full tank!","Second fuel","Price / L","Total expenses.","Volume","Full tank!" 2,"Third fuel","Price / L","Total expenses.","Volume","Full tank!"
> [fuel.zip](https://github.com/akhilrex/hammond/files/6598416/fuel.zip)

Whats the use case for 2nd and 3rd fuel here? Also what is the last column where some entries have "Trip" value?
